schema.json

```diff
@@ -276,6 +284,26 @@
         "os"
       ],
       "additionalProperties": false
+    },
+    "windows": {
+      "properties": {
+        "os": {
+          "type": "string"
+        },
+        "distribution": {
+          "const": "windows"
```

Comment:
Could we omit `distribution` and `versions`? I'm not sure - maybe it would require changes in the schema generator for that to work. But at least for now, there's not a reason to specify distribution or version for Windows yet.

Reply:
Required a few changes but all set!